### PR TITLE
CLOUDP-82836: Parse service version

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Parse service version header `X-MongoDB-Service-Version`

_Jira ticket:_ [CLOUDP-82836](https://jira.mongodb.org/browse/CLOUDP-82836)

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

⚠️ this change depends on https://github.com/mongodb/go-client-mongodb-atlas/pull/248 ⚠️

### CODE REVIEW

**Review by colm-quinn — APPROVED** [review]
LGTM

File: opsmngr/opsmngr.go [hunk 1]
```diff
@@ -94,6 +96,7 @@ type Client struct {
 	ServerUsage            ServerUsageService
 	ServerUsageReport      ServerUsageReportService
 	LiveMigration          LiveDataMigrationService
+	ServiceVersion         ServiceVersionService
```
Comment (gssbzn) [inline opsmngr/opsmngr.go:99]: [q] do you need to duplicate the implementation? if the service is the same we do 
```suggestion
	ServiceVersion         atlas.ServiceVersionService
```

File: opsmngr/opsmngr.go [hunk 2]
```diff
@@ -289,6 +293,29 @@ func (c *Client) NewGZipRequest(ctx context.Context, method, urlStr string) (*ht
 	return req, nil
 }
 
+// NewPlainRequest creates an API request that accepts plain text.
+// A relative URL can be provided in urlStr, which will be resolved to the
+// BaseURL of the Client. Relative URLS should always be specified without a preceding slash.
+func (c *Client) NewPlainRequest(ctx context.Context, method, urlStr string) (*http.Request, error) {
```
[the remark this answers is not in the document]
Comment (fmenezes) [reply opsmngr/opsmngr.go:299]: It is actually used, the ServiceVersionService depends on it, I'll add the tests

**Review by gssbzn — APPROVED** [review]
LGTM thanks for the tests

**Review by colm-quinn — APPROVED** [review]
LGTM :)

**Pull request merged**